wolox-books/src/app/button/button.component.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../scss/vbles/colors';
+@import '../../scss/vbles/generic-styles';
```

Comment:
It would had been better to be aware of this in the layout training part, because that's where it comes from 😅but you're right, I thought "variables" as external portions of anything.. in the next commit will be outside of it 👍